tests/test_openclhpp.cpp

```diff
@@ -75,6 +75,7 @@ static cl::Program programPool[POOL_MAX];
 #if defined(cl_khr_command_buffer)
 static cl::CommandBufferKhr commandBufferKhrPool[POOL_MAX];
 #endif
+static cl::Device devicePool[POOL_MAX];
```

Comment:
This comment a few lines above makes me a little nervous:

```c
/* Pools of pre-allocated wrapped objects for tests. There is no device pool,
 * because there is no way to know whether the test wants the device to be
 * reference countable or not.
 */
```

Is this still relevant?  Do we really need a `devicePool`?

Reply:
@bashbaug Thanks for reply.
Comment just updated- because we have now devicePool and we need it.